linux/README.md

```diff
@@ -83,6 +83,26 @@ Nightly builds upload the most recent new master build to https://downloads.keym
 Jenkins now continuously builds Debian packages on every commit to master
 Periodically test packages will be uploaded to https://launchpad.net/~keymanapp/+archive/ubuntu/keyman-daily
 
+#### Launchpad
```

Comment:
A link to https://help.launchpad.net/Packaging/PPA/Uploading somewhere in this would be helpful for  troubleshooting and setting up for dput upload